war/pom.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why? Please revert.

Reply:
lord knows..  I edited this file via eclipse dependency panel and not by hand.